.github/workflows/publish.yml

```
@@ -22,50 +21,80 @@ jobs:
-            - name: Install poetry
-              run: pipx install poetry
+                ref: ${{ github.event.inputs.commit_sha || github.ref_name }}
 
             - name: Set up Python
               uses: actions/setup-python@v5
               with:
                 cache: 'poetry'
 
+            - name: Bump RC number from PyPI
+              id: bump_rc
+              run: |
```

actionlint (1.7.8)

property "pr" is not defined in object type {}

(expression)